specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/virtualMachineScaleSet.json

```diff
@@ -6534,6 +6599,34 @@
         "name": "ZonalPlatformFaultDomainAlignMode",
         "modelAsString": true
       }
+    },
+    "VMScaleSetScaleOutInput": {
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Specifies the number of virtual machines in the scale set."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
```

Comment:
why are we flattening it? is this necessary?

Reply:
I don't believe that its necessary, but I set it for consistency as I see that it's set for most VMSS objects. Do you see this as a concern or suggest otherwise?

Reply:
I think it will be cleaner without the flattening from API definition perspective. 

Is there any flattening happening in the backend? 
since there is just 1 property "zone" right now, it is the same where there is flattening or not

Reply:
Ahh, thanks! As discussed, removing the flattening to keep properties in its own section.